### PR TITLE
Update git version

### PR DESCRIPTION
**Pull request by scotthain**
Updated pinnings and git version.

@chef/engineering-services

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -14,8 +14,14 @@ provisioner:
   require_chef_omnibus: latest
 
 platforms:
+  - name: centos-5.11
+    run_list: yum-epel::default
   - name: centos-6.7
     run_list: yum-epel::default
+  - name: freebsd-9.3
```
Comment (yzl) [inline .kitchen.yml]: centos-7?

**Comment by yzl** [comment]
:+1:  (I don't consider my centos-7 comment a blocker.)

**Comment by schisamo** [comment]
IMO we should nuke the `Gemfile.lock` file in this repo.

**Comment by scotthain** [comment]
@schisamo Great idea - I was leaning that direction.

**Pull request merged**